src/HH4b/processors/objects.py

```diff
@@ -307,8 +352,7 @@ def vbf_jets(
     muons = muons[muons.pt > muon_pt]
 
     ak4_sel = (
-        jets.isTight
```

Comment:
so this is removed because we always require `good_ak4jets` that includes the correct tight jet id?

Reply:
Yes.